lib/mhv/account_creation/service.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'mhv/account_creation/configuration'
+
+module MHV
+  module AccountCreation
+    class Service < Common::Client::Base
+      configuration Configuration
+
+      def create_account(icn:, email:, tou_status:, tou_version:, tou_occurred_at:)
```

Comment:
So the tou values for this API are kinda a lie. Basically they only accept hardcoded values and don't have a purpose for anything other than the expected constant (except the tou_time). I think this Service should probably just take `icn` and `email`, and then grab the current ToU from the `UserAccount`. If it's not accepted then raise error, otherwise write the constants that MHV expects, and use the proper `occurred_at` time

Reply:
(My original proposal was for ToU settings to not be part of this API at all, maybe a check on their backend, but I guess this is what everyone stuck with)

Reply:
yeah, i debated just using the latest agreement but usually `lib` doesn't have "knowledge" of code in `app` but idk how strictly that is followed anyways

Reply:
Oh I see, that's fair. I could see it staying how it is then I think